examples/tutorials/date_time_charts.py

```diff
@@ -0,0 +1,349 @@
+# as an ``np.array`` object. Similar to :class:`xarray.DataArray` this wraps the
+# dataset before passing it as a paramater. However, ``np.array`` objects use less
+# memory and allow developers to specify datatypes, and even pass multiple.
+#
+
+x = np.array(["2010-06-01", "2011-06-01T12", "2012-01-01T12:34:56"], dtype="datetime64")
+y = [2, 7, 5]
+
+fig = pygmt.Figure()
+fig.plot(
+    projection="X10c/10c",
+    region=[datetime.datetime(2010, 1, 1), datetime.datetime(2012, 6, 1), 0, 10],
+    frame=["WS", "ag"],
+    x=x,
+    y=y,
+    style="s0.5c",
+    pen="1p",
+    color="blue",
+)
+fig.show()
+
+########################################################################################
+# Generating Region
+# -----------------
+#
+# Another way of creating charts involving datetime data can be done
+# by automatically generating the region of the plot. This can be done
+# by passing the dataframe to :meth:`pygmt.info`, which will find
+# maximum and minimum values for each column and create a list
+# that could be passed as region. Additionally, a spacing argument
+# can be passed to increase the range past the maximum and minimum
+# data points.
+
+data = [
+    ["20200712", 1000],
+    ["20200714", 1235],
+    ["20200716", 1336],
+    ["20200719", 1176],
+    ["20200721", 1573],
+    ["20200724", 1893],
+    ["20200729", 1634],
+]
+df = pd.DataFrame(data, columns=["Date", "Score"])
+df.Date = pd.to_datetime(df["Date"], format="%Y%m%d")
+
+fig = pygmt.Figure()
+region = pygmt.info(
+    table=df[["Date", "Score"]], per_column=True, spacing=(700, 700), coltypes="T"
+)
+
+fig.plot(
+    region=region,
+    projection="X15c/10c",
+    frame=["WSen", "afg"],
+    x=df.Date,
+    y=df.Score,
+    style="c0.4c",
+    pen="1p",
+    color="green3",
+)
+
+fig.show()
+
+########################################################################################
+# Setting Primary and Secondary Time Axes
+# ---------------------------------------
+#
+# This example focuses on labeling the axes and setting intervals
+# at which the labels are expected to appear. All of these modifications
+# are added to the ``frame`` parameter and each item in that list modifies
+# a specific section of the plot.
+#
+# Starting off with ``WS``, adding this string means that only
+# Western/Left (**W**) and Southern/Bottom (**S**) borders of
+# the plot will be shown. For more information on this, please
+# refer to :doc:`frame instructions </tutorials/frames>`.
+#
+# The other important item in the ``frame`` list is
+# ``"sxa1Of1D"``. This string modifies the secondary
+# labeling (**s**) of the x-axis (**x**). Specifically,
+# it sets the main annotation and major tick spacing interval
+# to one month (**a1O**) (capital letter o, not zero). Additionally,
+# it sets the minor tick spacing interval to 1 day (**f1D**).
+# The labeling of this axis is also modified using
+# ``pygmt.config(FORMAT_DATE_MAP="o")`` to use the month's
```

Comment:
May want to link to https://docs.generic-mapping-tools.org/latest/gmt.conf.html#term-FORMAT_DATE_MAP somehow.

```suggestion
# The labeling of this axis can be modified by setting
# :gmt-term:`FORMAT_DATE_MAP` to 'o' to use the month's
```